### ISSUE TITLE
Unable to guess Mrs.Fletcher.S01E02.Free.Sample.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb.mkv

### ISSUE THREAD

**Issue by Rouzax** [guessit-io/guessit#624]
I'm running Medussa and that is using Guessit to detect episodes which had an issue with an episode because it has Sample in the title.
The file is actually there
```
Run on:                 do 14-11-2019 at 13:51:21,98
Handling Category:      TV 
Torrent Name:           Mrs.Fletcher.S01E02.Free.Sample.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb.mkv
File Name Torrent       = "Mrs.Fletcher.S01E02.Free.Sample.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb"
Destination file        = "Mrs.Fletcher.S01E02.Free.Sample.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb.mkv"
File extension          = ".mkv"
```
**Medusa (please complete the following information):**

Medusa Info: | Branch: masterCommit: 80b323eefca7c8d3c233ded9f389b0af35725f17Version: 0.3.6Database: 44.14
-- | --
Python Version: | 3.7.4 (tags/v3.7.4:e09359112e, Jul 8 2019, 20:34:20) [MSC v.1916 64 bit (AMD64)]
SSL Version: | OpenSSL 1.1.1c 28 May 2019
OS: | Windows-10-10.0.17763-SP0
Locale: | nl_NL.cp1252

**Debug logs (at least 50 lines):**

<details>

```
2019-11-14 13:53:29 DEBUG	Thread-3 :: [80b323e] Broken providers already updated in the last hour
2019-11-14 13:51:51 INFO	TORNADO :: [80b323e] Post-processing completed.
2019-11-14 13:51:51 DEBUG	TORNADO :: [80b323e] No processable items found in folder: C:\TEMP\Torrent\PROCD\TV\Mrs.Fletcher.S01E02.Free.Sample.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb
2019-11-14 13:51:51 DEBUG	TORNADO :: [80b323e] Processing path: C:\TEMP\Torrent\PROCD\TV\Mrs.Fletcher.S01E02.Free.Sample.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb
2019-11-14 13:51:51 DEBUG	TORNADO :: [80b323e] API :: postprocess: {'force_replace': '1', 'is_priority': '1', 'delete_files': '1', 'path': 'C:\\TEMP\\Torrent\\PROCD\\TV\\Mrs.Fletcher.S01E02.Free.Sample.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb', 'return_data': '1', 'process_method': 'move', 'type': 'manual'}
2019-11-14 13:51:51 DEBUG	TORNADO :: [80b323e] API :: all kwargs: "{'cmd': 'postprocess', 'force_replace': '1', 'is_priority': '1', 'delete_files': '1', 'path': 'C:\\\\TEMP\\\\Torrent\\\\PROCD\\\\TV\\\\Mrs.Fletcher.S01E02.Free.Sample.1080p.AMZN.WEB-DL.DDP5.1.H.264-NTb', 'return_data': '1', 'process_method': 'move', 'type': 'manual'}"
2019-11-14 13:51:51 DEBUG	TORNADO :: [80b323e] API :: all args: ()
2019-11-14 13:51:51 DEBUG	TORNADO :: [80b323e] API :: 127.0.0.1 - gave correct API KEY. ACCESS GRANTED
```
</details>

**Comment by pawamoy** [comment]
TLDR: `guessit` detected `Free Sample` as the episode title. The filename did not contain any other episode title. Seems hard to guess right unless `Free.Sample` is explicitly blacklisted from title candidates or something.

**Comment by Rouzax** [comment]
I figured that as well, just wanted to make sure I reported it.

**Comment by ratoaq2** [comment]
This is a medusa issue. The information from guessit is pretty good for medusa to process it. Most likely it's medusa that blacklist any file with sample in the name

**Comment by Rouzax** [comment]
Medusa will pick this up, thanks.
